android/source/VideoLocker/src/org/edx/mobile/view/FindCoursesActivity.java

```diff
@@ -0,0 +1,65 @@
+package org.edx.mobile.view;
+
+import android.os.Bundle;
+import android.view.Menu;
+import android.view.MenuItem;
+import org.edx.mobile.R;
+import org.edx.mobile.base.BaseFragmentActivity;
+import org.edx.mobile.util.AppConstants;
+import org.edx.mobile.util.NetworkUtil;
+
+public class FindCoursesActivity extends BaseFragmentActivity {
+
+    @Override
+    protected void onCreate(Bundle savedInstanceState) {
+        super.onCreate(savedInstanceState);
+        setContentView(R.layout.activity_find_courses);
+
+        // configure slider layout. This should be called only once and
+        // hence is shifted to onCreate() function
+        configureDrawer();
+
+        try{
+            segIO.screenViewsTracking(getString(R.string.find_courses_title));
+        }catch(Exception e){
+            logger.error(e);
+        }
+
+        if (!(NetworkUtil.isConnected(this))) {
+            AppConstants.offline_flag = true;
```

Comment:
Removed this line for now. Will reintroduce when the UI for offline mode is finalized.